### DISCUSSION TITLE
Self hosted prisma1 cloud question

### DISCUSSION THREAD

**Discussion by benliger1987** [prisma/prisma#14409] [Q&A]
Hello,

I just successfully hosted prisma1 cloud in a Digital Ocean droplet as the prisma cloud is due to be sunset soon. I have a couple of questions:

1. I host my prisma1 databases on Heroku. Once the prisma cloud does sunset - there is nothing I need to change in my applications code right? ie The prisma cloud is simple an extension for managing entries through a GUI or Graphql playground, correct?
2. I am generating a Bearer token via `prisma token` to connect to my project in Graphql playground by adding the token in Graphql Playgrounds HTTP Headers box like so:

```
{
  "Authorization":"Bearer ey...."
}
```

This works fine for the playground but when I want to access the GUI of prisma cloud by visiting https://my-hosted-prisma-cloud.com/my-project/prod/_admin?headers={"Authorization":"Bearer%20ey...."} I get a message saying:

`Connection unsuccessful, Incorrect access token provided`

Why would that be?

**Comment by andrewicarlson** [comment]
@benliger1987 you'll need to verify that you can deploy to your new Digital Ocean droplet without being logged in to Prisma 1 Cloud. To do so you need to set a `PRISMA_MANAGEMENT_API_SECRET` in your local environment. [You can learn more about that here](https://v1.prisma.io/docs/1.34/faq/migrate-to-self-hosted-fq15/#step-1:-identify-features).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
